### PR TITLE
Caught up with the EU extensions (closes #18)

### LINKED ISSUE #18: Update GPA mappings to match new OCDS for EU extensions
* [x] Place of performance
* [x] Options

There might be other mappings to update.

### PR DESCRIPTION
**Pull request by ColinMaudry**
- I was tempted to add guidance for data not explicitly mentioned in GPA, for instance URL and description of the electronic auction, but I stuck to the text
- not sure about entering "negotiated" in `procurementMethodDetails` (since `procurementMethodModalities` doesn't exist anymore
- not sure either about jumping to the conclusion that VII:2(k) is about framework agreements

### CODE REVIEW

File: docs/gpa.md [hunk 1]
```diff
@@ -107,10 +108,10 @@ The addition of `period` to the `Milestone` building block is [under discussion]
         <td markdown=1>
 
 * Use 'open', 'selective' or 'limited' for *the procurement method that will be used* in `tender/procurementMethod`
-* If *it will involve negotiation*, add 'negotiated' to `tender/procurementMethodModalities`
-* If *it will involve electronic auction*, add 'electronicAuction' to `tender/procurementMethodModalities`
+* If *it will involve negotiation*, add 'negotiated' to `tender/procurementMethodDetails`
```
Comment (jpmckinney) [inline docs/gpa.md:111]: Use double quotes for text (single quotes for codes).

For the EU, we have one case of setting it to "Competitive procedure with negotiation", so I think "negotiated" here is fine.

File: docs/gpa.md [hunk 2]
```diff
@@ -157,7 +158,14 @@ This requires the [Additional Contact Points](https://github.com/open-contractin
         <td>where, pursuant to Article IX, a procuring entity intends to select a limited number of qualified suppliers to be invited to tender, the criteria that will be used to select them and, where applicable, any limitation on the number of suppliers that will be permitted to tender; and</td>
         <td markdown=1>
 
-* Add this to `tender/eligibilityCriteria`
+* Set `tender.techniques.hasFrameworkAgreement` to `true`
+* Enter *the number of suppliers that will be permitted to tender* in `tender.techniques.frameworkAgreement.maximumParticipants`
```
Comment (jpmckinney) [inline docs/gpa.md:162]: Use https://github.com/open-contracting-extensions/ocds_secondStageDescription_extension/blob/master/release-schema.json instead.

File: docs/gpa.md [hunk 3]
```diff
@@ -157,7 +158,14 @@ This requires the [Additional Contact Points](https://github.com/open-contractin
         <td>where, pursuant to Article IX, a procuring entity intends to select a limited number of qualified suppliers to be invited to tender, the criteria that will be used to select them and, where applicable, any limitation on the number of suppliers that will be permitted to tender; and</td>
         <td markdown=1>
 
-* Add this to `tender/eligibilityCriteria`
+* Set `tender.techniques.hasFrameworkAgreement` to `true`
```
Comment (jpmckinney) [inline docs/gpa.md:161]: Remove.

File: docs/gpa.md [hunk 4]
```diff
@@ -157,7 +158,14 @@ This requires the [Additional Contact Points](https://github.com/open-contractin
...
+* Enter *the number of suppliers that will be permitted to tender* in `tender.techniques.frameworkAgreement.maximumParticipants`
+
+This requires the [Techniques](https://github.com/open-contracting-extensions/ocds_techniques_extension) extension.
+
+* Add *the criteria that will be used to select [the suppliers]* to `tender.selectionCriteria.criteria`
```
Comment (jpmckinney) [inline docs/gpa.md:166]: Let's give both options for structured and unstructured data. Unstructured data can simply use `tender.selectionCriteria.description`.

File: docs/gpa.md (hunk 2 above)
Comment (ColinMaudry) [reply docs/gpa.md:162]: :facepalm: :+1:

File: docs/gpa.md (hunk 1 above)
Comment (ColinMaudry) [reply docs/gpa.md]: Done

File: docs/gpa.md (hunk 2 above)
Comment (ColinMaudry) [reply docs/gpa.md]: Done

File: docs/gpa.md (hunk 3 above)
Comment (ColinMaudry) [reply docs/gpa.md]: Done

File: docs/gpa.md (hunk 4 above)
Comment (ColinMaudry) [reply docs/gpa.md]: Done

**Pull request merged**